package.json

```diff
@@ -1,4 +1,6 @@
 {
+  "name": "gulp-babel-minify",
+  "version": "0.2.0",
```

Comment:
these are not required

Reply:
I removed the unnecessary commits to the package.json file.